### PR TITLE
[integration] fix: check running containers

### PR DESCRIPTION
**Pull request by arrabito**
BEGINRELEASENOTES

*tests
FIX: check running containers in integration tests script

ENDRELEASENOTES

### CODE REVIEW

**Comment by arrabito** [comment]
At least on MacOS platform:

`./integration_tests.py`
fails if there are no running containers with the following error:

```
CalledProcessError: Command '['docker', 'compose', '-f', 
PosixPath('/var/folders/7j/qdb2nq852973tsy1_v99qzrm0000gq/T/tmp9hv6jxlc/ci/docker-compose.yml'), 'ps', '-q', '-a']' returned non-zero exit 
status 1.
```
Disabling the exit code check makes the script works on all platforms.

**Comment by maxnoe** [comment]
I ran into the same issue on Ubuntu 22.04 with a fresh checkout.

File: integration_tests.py [hunk 1]
```diff
@@ -593,7 +593,6 @@ def _check_containers_running(*, is_up=True):
             ["docker", "compose", "-f", docker_compose_fn, "ps", "-q", "-a"],
```
Comment (chrisburr) [inline integration_tests.py:593]: Can you try this instead?

```suggestion
            ["docker", "compose", "-f", str(docker_compose_fn), "ps", "-q", "-a"],
```

Comment (arrabito) [reply integration_tests.py:593]: I've tried but I get the same error.

Comment (maxnoe) [reply integration_tests.py:593]: The error is not a python error.

docker compose ps just has a non-zero exit code when no containers are running

File: integration_tests.py [hunk 2]
```diff
@@ -593,7 +593,6 @@ def _check_containers_running(*, is_up=True):
...
             stdout=subprocess.PIPE,
             env=_make_env({}),
-            check=True,
             text=True,
```
Comment (chrisburr) [inline integration_tests.py:598]:
```suggestion
            # docker compose ps has a non-zero exit code when no containers are running
            check=False,
            text=True,
```

**Review by chrisburr — APPROVED** [review]

**Pull request merged**